ios/src/TGGeoPolygon.mm

```diff
@@ -0,0 +1,80 @@
+//
+//  TGGeoPolygon.mm
+//  tangram
+//
+//  Created by Karim Naaji on 10/27/16.
+//
+//
+
+#import "TGGeoPolygon.h"
+
+#include <vector>
+
+@interface TGGeoPolygon () {
+    std::vector<TGGeoPoint> coordinates;
+    std::vector<int> rings;
```

Comment:
That would be `NSMutableArray` but it can't hold `int` so you would have to convert to using `NSNumber` which is probably not something you'd want. You could always get fancy with Objective-C++ and use C++ arrays like `int *rings = new int[...]` but given how we're using them that might not be advantageous. Also I can't remember if that style of dynamic array ensures contiguous memory allocation.